### PR TITLE
test: bump test-run to new version

### PR DESCRIPTION
**Pull request by Serpentian**
Bump test-run to new version with the following improvements:

- luatest: fix ability to run a test several times [1]
- Enable luatest logging [2]
- tap13: fix parsing non-utf8 chars [3]

[1] tarantool/test-run@240cdea
[2] tarantool/test-run@b8b60b4
[3] tarantool/test-run@7290540

NO_DOC=test
NO_TEST=test
NO_CHANGELOG=test

### CODE REVIEW

**Comment by Serpentian** [comment]
@ylobankov it's really tedious, that `yes` doesn't work in tarantool. Let's bump `test-run`

**Comment by ylobankov** [comment]
> @ylobankov it's really tedious, that `yes` doesn't work in test-run. Let's bump `test-run`

@ochaplashkin is working on it.

File: test/box-luatest/gh_7904_export_box_schema_version_to_public_api_test.lua [hunk 1]
```diff
@@ -23,7 +23,8 @@ g.test_box_internal_schema_version_deprecation = function(cg)
     local deprecation_warning =
         'box.internal.schema_version will be removed, please use box.info.schema_version instead'
     t.assert_is_not(cg.server:grep_log(deprecation_warning, 256), nil)
-    local log_file = g.server:exec(function() return box.cfg.log end)
+    local log_file = g.server:exec(function()
+        return rawget(_G, 'box_cfg_log_file') or box.cfg.log end)
```
Comment (ylobankov) [inline test/box-luatest/gh_7904_export_box_schema_version_to_public_api_test.lua:27]: Please move `end)` to the new line.

Comment (ochaplashkin) [reply test/box-luatest/gh_7904_export_box_schema_version_to_public_api_test.lua]: Fixed :rocket:

**Comment by sergepetrenko** [comment]
@ylobankov, all the tests pass. Can we merge this?

**Review by ylobankov — APPROVED** [review]

**Comment by ylobankov** [comment]
> @ylobankov, all the tests pass. Can we merge this?

I am OK.

File: test/box-luatest/gh_7904_export_box_schema_version_to_public_api_test.lua (hunk 1 above)
Comment (sergepetrenko) [reply test/box-luatest/gh_7904_export_box_schema_version_to_public_api_test.lua]: Resolving since the PR is approved.

**Comment by sergepetrenko** [comment]
Backports:
3.1 https://github.com/tarantool/tarantool/pull/10298
2.11 https://github.com/tarantool/tarantool/pull/10299

**Pull request merged**